frontend/views/top_containers/show.html.erb

```diff
@@ -33,7 +33,21 @@
           <%= readonly.label_and_textarea "exported_to_ils", :default => I18n.t("top_container.not_exported") %>
 
           <%= readonly.label_and_boolean "restricted" %>
+          <%= readonly.label_and_boolean "override_restricted" %>
 
```

Comment:
We have a "Restricted?" column on the index page too.  Should that be `override_restricted || restricted`?